### PR TITLE
Add variable name to attrition table

### PR DESCRIPTION
**Pull request by pieterlukasse**
Jira Ticket: [VADC-31](https://ctds-planx.atlassian.net/browse/VADC-31)

### Improvements

- attrition table now contains a user-defined name instead of an auto-generated one

### CODE REVIEW

File: tests/utils_tests/parsing_test.go [hunk 1]
```diff
@@ -49,10 +49,10 @@ func TestParsePrefixedConceptIdsAndDichotomousIds(t *testing.T) {
 	requestContext.Request = new(http.Request)
 	requestBody := "{\"variables\":[{\"variable_type\": \"concept\", \"concept_id\": 2000000324}," +
 		"{\"variable_type\": \"concept\", \"concept_id\": 2000000123}," +
-		"{\"variable_type\": \"custom_dichotomous\", \"cohort_ids\": [1, 3]}]}"
+		"{\"variable_type\": \"custom_dichotomous\", \"provided_name\": \"test\", \"cohort_ids\": [1, 3]}]}"
```
Comment (UchicagoZchen138) [inline tests/utils_tests/parsing_test.go:52]: can we add another custom diachotomous variable but with no provided_name to test out the auto generate name?

Comment (UchicagoZchen138) [reply tests/utils_tests/parsing_test.go:52]: actually nvm saw we're testing it in TestRetrieveAttritionTable

File: models/cohortdata.go [hunk 2]
```diff
@@ -30,6 +32,8 @@ type PersonIdAndCohort struct {
 	CohortId int64
 }
 
+// This function returns the subjects that belong to both cohorts (the intersection of both cohorts)
+// TODO - name this function as such
```
Comment (UchicagoZchen138) [inline models/cohortdata.go:36]: saw this TODO, is this just changing the function name? And if so should we just do it now?

**Comment by pieterlukasse** [comment]
> we should update readme

good catch. Done in last commit.

**Pull request merged**